.env.development

```diff
@@ -7,6 +7,7 @@ REACT_APP_REPORTING_API_ROOT=https://dev-k8s.treetracker.org/reporting
 REACT_APP_REGION_API_ROOT=https://dev-k8s.treetracker.org/regions
 REACT_APP_TREETRACKER_API_ROOT=https://dev-k8s.treetracker.org/treetracker
 REACT_APP_GROWER_QUERY_API_ROOT=https://dev-k8s.treetracker.org/grower-account-query
+REACT_APP_IMAGES_API_ROOT=https://dev-k8s.treetracker.org/images
```

Comment:
Let's include the corresponding test, staging and production URLs in their respective .env files so we don't need another PR once they're deployed.